src/assets/Articles.jsx

```diff
@@ -25,17 +27,23 @@ export default function Articles() {
   if (isLoading) return <p>Loading...</p>;
   if (isError) return <p>Something went wrong</p>;
 
-  return (
-    <div>
-      {articles.map((article) => {
-        return (
-          <Link key={article.article_id} to={`/articles/${article.article_id}`}>
-            <div>
-              <p>{article.title}</p>
-            </div>
-          </Link>
-        );
-      })}
-    </div>
-  );
+  if (Articles !== undefined) {
```

Comment:
you shouldn't need this check as you have checks above for if its loading or if theres an error